libraries/SuplaDevice/examples/Supla_MAX6675_No_SSL/Supla_MAX6675_No_SSL.ino

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright (C) AC SOFTWARE SP. Z O.O.
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+*/
+
+#include <SPI.h>
+#include <SuplaDevice.h>
+
+
+// Add include to MAX6675_K sensor
+#include <supla/sensor/MAX6675_K.h>
+
+#include <supla/network/esp_wifi.h>
+Supla::ESPWifi wifi("your_wifi_ssid", "your_wifi_password");
+
+void setup() {
+
+  Serial.begin(115200); 
+
+  // Replace the falowing GUID with value that you can retrieve from https://www.supla.org/arduino/get-guid
+  char GUID[SUPLA_GUID_SIZE] = {0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00};
+
+  // Replace the following AUTHKEY with value that you can retrieve from: https://www.supla.org/arduino/get-authkey
+  char AUTHKEY[SUPLA_AUTHKEY_SIZE] = {0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00};
+
+  /*
+   * Having your device already registered at cloud.supla.org,
+   * you want to change CHANNEL sequence or remove any of them,
+   * then you must also remove the device itself from cloud.supla.org.
+   * Otherwise you will get "Channel conflict!" error.
+   */
+
+  new Supla::Sensor::MAX6675_K(15, 13, 12);  //  MAX6675(pin_CLK, pin_CS, pin_DO)
+
+  /* By default this library uses secured connection with SUPLA server. You can switch it of by writing below line 
+   * Works only with ESP8266 boards!
+   */
+
+  wifi.enableSSL(false);
```

Comment:
remove it, and rename example to something like: "Thermometer_MAX6675" (don't use "Supla" name in examples - they are anyway showed by Arduino under SuplaDevice "folder".